src/literature/components/CollapsibleAbstract/CollapsibleAbstract.scss

```diff
@@ -0,0 +1,12 @@
+.__CollapsibleAbstract__ {
+  margin-top: 0.5rem;
+  .abstractContainer {
+    line-height: 1.2rem;
+    .__Latex__ .katex span {
```

Comment:
why did you need this hack? if it's really necessary and not a breaking change for other components that use `Latex`, we should move it to `Latex` component.

Reply:
I tried doing that. However, it messes up some titles.
![screenshot from 2018-07-05 21-51-38](https://user-images.githubusercontent.com/11242410/42344798-bbf374f6-809d-11e8-9375-f1a7c0586850.png)

Reply:
why did you need this in the first place?

Reply:
To display the 'More/Less' text right next to the abstract. (So, the class is applied to the span in abstract container)

Reply:
@drjova let's not put it inline if this is the only way, because I am sure this css will break some latex rendering.